src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -102,8 +111,16 @@ public DurableTaskExtension()
             IApplicationLifetimeWrapper hostLifetimeService,
             IDurableHttpMessageHandlerFactory durableHttpMessageHandlerFactory = null,
             ILifeCycleNotificationHelper lifeCycleNotificationHelper = null,
+#if !FUNCTIONS_V1

```

Comment:
small nit: we don't need to include `IMessageSerializerSettingsFactory messageSerializerSettingsFactory = null,` in the conditional compilation. 

Overall, very big improvement over previous iteration!

Reply:
Done